frontend/src/components/Button.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import styled from 'styled-components'
+
+interface StyledButtonProps {
+  primary?: boolean
+  danger?: boolean
+}
+
+const ButtonWrapper = styled.button`
+  background: ${(props: StyledButtonProps) =>
+    props.primary ? 'grey' : 'white'};
+  color: ${(props: StyledButtonProps) => (props.primary ? 'white' : 'grey')};
+  border-radius: 5px;
+  height: 3.125 em;
+  width: auto;
+  font-family: Cabin, sans-serif;
+  font-weight: bold;
+  align-items: center;
+`
```

Comment:
Other things to add:
- let's add left/right padding of 0.5em
- remove outline on focus
- add some kind of haptic feedback (i.e. on hover, change cursor to a pointer and add thin black border)
- add `fillWidth` prop; if passed in, have the button fill the width of its container; if not passed in, width is auto